### ISSUE TITLE
[BUG] cnf-testsuite no longer uses tools/helm when global helm not available

### ISSUE THREAD

**Issue by agentpoyo** [cncf/cnf-testsuite#1377]
**Describe the bug**
cnf-testsuite no longer uses the tools/helm binary to run tests when global helm is not found.

**To Reproduce**
Setup cnf-testsuite; make sure no global helm is available, run `workload` test against CNF. See error output on all tests that require helm.

**Expected behavior**
cnf-testsuite should use the tools/helm when global helm is not found or available by either using direct path to the binary or setting it in the PATH temporarily.

**Example Error Output**
This is a snippet running `workload`, all tests unless they skip fail to use the tools/helm that is available:
```
E, [2022-04-21 20:46:21 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-04-21 20:46:21 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/execution.cr:20:7 in '__crystal_main'
E, [2022-04-21 20:46:21 -04:00 #4124537] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/main.cr:110:5 in 'main'
E, [2022-04-21 20:46:21 -04:00 #4124537] ERROR -- cnf-testsuite: src/env/__libc_start_main.c:94:2 in 'libc_start_main_stage2'
E, [2022-04-21 20:46:21 -04:00 #4124537] ERROR -- cnf-testsuite: Global install of Helm not found
E, [2022-04-21 20:46:21 -04:00 #4124537] ERROR -- cnf-testsuite: /usr/share/crystal/src/regex/match_data.cr:149:7 in 'helm'
E, [2022-04-21 20:46:21 -04:00 #4124537] ERROR -- cnf-testsuite: /usr/share/crystal/src/log/main.cr:38:3 in 'generate_manifest_from_templates'
E, [2022-04-21 20:46:21 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/src/tasks/utils/cnf_manager.cr:100:23 in '->'
E, [2022-04-21 20:46:21 -04:00 #4124537] ERROR -- cnf-testsuite: /usr/share/crystal/src/primitives.cr:266:3 in 'all_cnfs_task_runner'
E, [2022-04-21 20:46:21 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/src/tasks/utils/task.cr:19:9 in 'task_runner'
E, [2022-04-21 20:46:21 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/state.cr:323:3 in '->'
E, [2022-04-21 20:46:21 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-04-21 20:46:21 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-04-21 20:46:21 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/execution.cr:20:7 in '__crystal_main'
E, [2022-04-21 20:46:21 -04:00 #4124537] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/main.cr:110:5 in 'main'
E, [2022-04-21 20:46:21 -04:00 #4124537] ERROR -- cnf-testsuite: src/env/__libc_start_main.c:94:2 in 'libc_start_main_stage2'
⏭️  SKIPPED: Mysql not installed 🧫
State final score: 0 of 20

E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: Global install of Helm not found
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /usr/share/crystal/src/regex/match_data.cr:149:7 in 'helm'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /usr/share/crystal/src/log/main.cr:38:3 in 'generate_manifest_from_templates'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/src/tasks/utils/cnf_manager.cr:100:23 in '->'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /usr/share/crystal/src/primitives.cr:266:3 in 'all_cnfs_task_runner'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/src/tasks/utils/task.cr:19:9 in 'task_runner'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/state.cr:211:3 in '->'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/execution.cr:20:7 in '__crystal_main'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/main.cr:110:5 in 'main'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: src/env/__libc_start_main.c:94:2 in 'libc_start_main_stage2'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: Global install of Helm not found
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /usr/share/crystal/src/regex/match_data.cr:149:7 in 'helm'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /usr/share/crystal/src/log/main.cr:38:3 in 'generate_manifest_from_templates'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/src/tasks/utils/cnf_manager.cr:100:23 in '->'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /usr/share/crystal/src/primitives.cr:266:3 in 'all_cnfs_task_runner'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/src/tasks/utils/task.cr:19:9 in 'task_runner'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/src/tasks/workload/security.cr:195:3 in '->'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /workspace/lib/sam/src/sam/execution.cr:20:7 in '__crystal_main'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: /usr/share/crystal/src/crystal/main.cr:110:5 in 'main'
E, [2022-04-21 20:46:24 -04:00 #4124537] ERROR -- cnf-testsuite: src/env/__libc_start_main.c:94:2 in 'libc_start_main_stage2'
/usr/share/crystal/src/regex/match_data.cr:149:7 in 'helm'
/workspace/lib/helm/helm.cr:221:12 in 'helm_repo_add'
/workspace/src/tasks/falco_setup.cr:24:7 in '->'
/workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
/workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
/workspace/lib/sam/src/sam/task.cr:44:29 in 'call'
/workspace/lib/sam/src/sam/execution.cr:20:7 in '__crystal_main'
/usr/share/crystal/src/crystal/main.cr:110:5 in 'main'
src/env/__libc_start_main.c:94:2 in 'libc_start_main_stage2'
Global install of Helm not found
```

**Comment by agentpoyo** [comment]
#1376 is a related ticket until it was discovered that none of the tests are now using the tools/helm binary downloaded during the `setup` step if no global helm is found.

**Comment by HashNuke** [comment]
This issue is resolved as apart of #1375. Moving this to peer review.

**Comment by agentpoyo** [comment]
This has been fixed or all tests would be failing. No AC required.
